docs/tools/offline/mbed-studio.md

```diff
@@ -0,0 +1,193 @@
+
+Once Mbed Studio has a binary of the Mbed OS program, it deploys that binary to the currently selected target and resets the target, so the program begins to run.
+
+To make it happen, click the green run icon to launch the default run configuration for your connected development board.
+
+Alternatively, open the context menu (right-click, ctrl-click) on the `mbed-os-example-blinky` project and select `Run As > Run Configurations...` from the menu. You can also use the menu by selecting `Run > Run Configurations` (both ways shown below).
+
+In the `Run Configurations` window, right-click on `Mbed Deploy`, and select `New` to create an `Mbed Deploy` run configuration. This creates the run configuration you need to populate with details of your project and target. For project (if everything isn’t already populated), click `Choose`, and select the `mbed-os-example-blinky` project. The binary Mbed Studio builds fills in automatically. If you used a different project name, the binary name reflects that. Finally, click the Refresh button to ensure that the current target board’s ID updates if it hasn’t already.
+
+After you have configured everything, click on the Apply button and then on the Run button.
+
+Watch the primary LED on your connected target begin to blink!
+
+#### Debugging
+
+Now that you have learned the basics of Mbed Studio, it is time to learn how to debug projects within the IDE. There is a simplified debug configuration specifically for easy debugging of Mbed OS projects. There are also additional configurations for more advanced debugging. This section covers the Mbed OS debug configuration, as well as the advanced PyOCD GDB debug configuration.
+
+##### Simple debugging
+
+To begin debugging, click the bug icon, which launches the default debug configuration.
+
+Alternatively, you can create the debug launch configuration and then begin debugging. To create a debug launch configuration, right-click on a project and select `Debug As... > Debug Configurations` from the context menu. (You can also use the menu bar as shown below.)
+
+Once created, the configuration prepopulates all the readily inferred values. Click `Choose...` next to the project textbox, and select the project you wish to debug if it isn’t already populated. After that, connect a board, and the project is ready to debug. The Mbed Debug launch configuration prepopulates the PyOCD GDB server and the GDB client to locations to the internal Mbed Studio tools. Select the Debug button in the lower right portion of the window to start debugging.
+
+Mbed Studio automatically switches into the Mbed Debug perspective when debugging with the Mbed Debug launch configuration. The project also breakes on a default break point. The resulting workspace looks like the image below.
```

Comment:
Note: There is no image below. Please add the image to the "images" folder at the top level of these docs, and include the AWS link in this .md file.

Reply:
Ah yes, the original docs have lots of screenshots in them, which I removed for ease of updating this in future. I can either add the images back in, or delete any mention of them.